$OpenBSD$
Allow path to Python interpreter to be overwritten via make flags.
--- qtwebkit/Source/JavaScriptCore/DerivedSources.make.ports.orig	Wed Nov  5 00:00:03 2014
+++ qtwebkit/Source/JavaScriptCore/DerivedSources.make	Wed Nov  5 00:03:15 2014
@@ -33,6 +33,8 @@ VPATH = \
     $(JavaScriptCore)/jit \
 #
 
+PYTHON =	python
+
 .PHONY : all
 all : \
     ArrayConstructor.lut.h \
@@ -72,12 +74,12 @@ docs/bytecode.html: make-bytecode-docs.pl Interpreter.
 # character tables for Yarr
 
 RegExpJitTables.h: create_regex_tables
-	python $^ > $@
+	$(PYTHON) $^ > $@
 
 KeywordLookup.h: KeywordLookupGenerator.py Keywords.table
-	python $^ > $@
+	$(PYTHON) $^ > $@
 
 # udis86 instruction tables
 
 udis86_itab.h: $(JavaScriptCore)/disassembler/udis86/itab.py $(JavaScriptCore)/disassembler/udis86/optable.xml
-	(PYTHONPATH=$(JavaScriptCore)/disassembler/udis86 python $(JavaScriptCore)/disassembler/udis86/itab.py $(JavaScriptCore)/disassembler/udis86/optable.xml || exit 1)
+	(PYTHONPATH=$(JavaScriptCore)/disassembler/udis86 $(PYTHON) $(JavaScriptCore)/disassembler/udis86/itab.py $(JavaScriptCore)/disassembler/udis86/optable.xml || exit 1)
